src/views/Warehouse/Stock.tsx

```diff
@@ -0,0 +1,5 @@
+import { Heading } from '@chakra-ui/react';
+
+const Stock = () => <Heading>MOCK: Zapasy</Heading>;
```

Comment:
It should be `Stocks` as the rest with ending `s`.

Reply:
I relied on this [resource](https://www.wordhippo.com/what-is/the-plural-of/stock.html#:~:text=The%20noun%20stock%20can%20be,or%20a%20collection%20of%20stocks.), but I changed it.